### ISSUE TITLE
How to set OnClickItemListener

### ISSUE THREAD

**Issue by tellfa** [nanyuweiyi/WheelView#1]
I need to OnClickItemListener method, how can i it?

**Comment by tellfa** [comment]
Please add this listener into you library and update you library.
please soon, because i really need this feature

**Comment by nanyuweiyi** [comment]
你想增加点击事件建议用recycleview，滚轮事件一般用于选择。

**Comment by tellfa** [comment]
Please write English my friend. can you add this feature to your great library? please

**Comment by nanyuweiyi** [comment]
please use it:
        loopView.setListener(new OnItemSelectedListener() {
            @Override
            public void onItemSelected(int index) {
                if (toast == null) {
                    toast = Toast.makeText(MainActivity.this, "item " + index, Toast.LENGTH_SHORT);
                }
                toast.setText("item " + index);
                toast.show();
            }
        });

**Comment by tellfa** [comment]
yes bro, in this code item item show (in center) show Toast . But i want just when click on item show me Toast.
how can i it? 
can you add this into your library? please
